app/src/main/java/com/neiapp/spocan/models/Initiative.java

```diff
@@ -28,8 +28,14 @@
     private String nickname;
     private LocalDateTime date;
     private InitiativeStatus status;
+    private String reject_motive;
 
-    public Initiative(String _id, InitiativeStatus status, String description, String image, String nickname, String dateStrUTC) {
+    public Initiative(String _id, InitiativeStatus status, String description, String image, String nickname, String dateStrUTC, String reject_motive) {
+        if(reject_motive!=null){
+            this.reject_motive = reject_motive;
+        }else{
+            this.reject_motive = null;
+        }
```

Comment:
Esto no es necesario, podés directamente hacer this.reject_motive = reject_motive. Si el argumento es nulo se inicializa en nulo, igual a lo que resuelve el if.